### PR TITLE
Added organization value

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

File: chinese/wg-value/organizational-value/labor-investment.md [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
+- 贡献数量
+- 按贡献者类型（内部/外部）划分的贡献数量
+- 按贡献类型（如提交、问题、拉取请求）划分的贡献数量
+
+参数包括：
+
+- 每小时劳动率
+- 创建贡献的平均劳动时间（按照贡献类型分类）
+
+人力投资 = 每一种贡献类型的总和（贡献数量 * 创造贡献的平均工时 * 平均每小时费率）
```
Comment (king-gao) [inline chinese/wg-value/organizational-value/labor-investment.md:33]: Average hourly rate: can we translate into "劳动生产率" or “平均每小时劳动率”instead of“平均每小时费率”

**Review by king-gao — COMMENTED** [review]
hi @zhongjun2 , thank you for translating the doc. you did really awesome job, I have commented for your PR, can you check it?

File: chinese/wg-value/organizational-value/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# 人力投资
+
+目标：从组织的角度看项目是否具有经济价值。
+
+| 关注领域 | 目标 |
+| --- | --- |
+| [人力投资](labor-investment.md) | 如何计算组织投入的员工所做的贡献（例如：提交、问题和请求）的成本是多少？ |
```
Comment (eyehwan) [inline chinese/wg-value/organizational-value/README.md:7]: 提交、问题和请求 ->代码提交，议题和更改请求

File: chinese/wg-value/organizational-value/labor-investment.md [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
-
-
-## 参考资料
-
-- [启动开源计划办公室](https://www.slideshare.net/caniszczyk/starting-an-open-source-program-office-ospo)
-- [创办开源计划办公室](https://events19.linuxfoundation.org/wp-content/uploads/2018/07/OSLS_2019-untold-story-of-OSPO.pdf)
-- [企业开源](https://d1.awsstatic.com/Open%20Source/enterprise-oss-book.pdf)
+# 人力投资
+
+问题：组织投入人力对社区所做的贡献（例如：评论、问题、代码提交）花费的成本是多少
```
Comment (eyehwan) [inline chinese/wg-value/organizational-value/labor-investment.md:3]: 评论、问题、代码提交-》代码提交，议题和更改请求

File: chinese/wg-value/organizational-value/labor-investment.md [hunk 4]
```diff
@@ -1,51 +1,52 @@
...
+
+## 描述
+
+开源项目通常由组织的人力投入来支撑。该指标跟踪组织对单个项目的经济投入（体现在人工成本）。
```
Comment (eyehwan) [inline chinese/wg-value/organizational-value/labor-investment.md:7]: 人工成本-> 人力成本

File: chinese/wg-value/organizational-value/labor-investment.md [hunk 5]
```diff
@@ -1,51 +1,52 @@
...
+
+## 目标
+
+随着组织参与度对开源项目变得越来越重要，组织必须清楚了解其人力投资。该指标的目的是为从事开源项目的组织提高人力成本的透明度。该指标给开源项目办公室（OSPO）管理员提供了一种通过项目投资组合比较人力成本的方法。比如：人力投资指标能用在确定投资的优先顺序或者确定投资回报。例如：
```
Comment (eyehwan) [inline chinese/wg-value/organizational-value/labor-investment.md:11]: 管理员-》经理

File: chinese/wg-value/organizational-value/labor-investment.md [hunk 6]
```diff
@@ -1,51 +1,52 @@
...
+  * 以人力投资反应和比较开源贡献与内部工作的人力成本
+  * 以人力投资比较项目组合的项目效益
+
+## 实现
+
+基础指标包括：
+
+- 贡献数量
+- 按贡献者类型（内部/外部）划分的贡献数量
+- 按贡献类型（如提交、问题、拉取请求）划分的贡献数量
```
Comment (eyehwan) [inline chinese/wg-value/organizational-value/labor-investment.md:26]: 代码提交，议题和更改请求

**Comment by eyehwan** [comment]
Amazing work! Thanks a lot @zhongjun2 ！

File: chinese/wg-value/organizational-value/labor-investment.md (hunk 1 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/labor-investment.md:33]: Done

File: chinese/wg-value/organizational-value/README.md (hunk 2 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/README.md:7]: Done

File: chinese/wg-value/organizational-value/labor-investment.md (hunk 3 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/labor-investment.md:3]: Done

File: chinese/wg-value/organizational-value/labor-investment.md (hunk 4 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/labor-investment.md:7]: Done

File: chinese/wg-value/organizational-value/labor-investment.md (hunk 5 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/labor-investment.md:11]: Done

File: chinese/wg-value/organizational-value/labor-investment.md (hunk 6 above)
Comment (zhongjun2) [reply chinese/wg-value/organizational-value/labor-investment.md:26]: Done

**Pull request merged**